### PR TITLE
Move Mythical pokemon down in VGC teambuilder

### PR DESCRIPTION
**Pull request by HoeenCoder**
This should in theory work, but it didnt apply when I tested it via testclient.html. PRed anyways in case its something that needs to be applied main's (build script for teambuilder probably).

### CODE REVIEW

**Comment by Zarel** [comment]
The reason Mythical is first is so it's easy to remove. Putting it last makes the removal a bit more complicated.

**Comment by Zarel** [comment]
And also doesn't actually remove them...

**Comment by Zarel** [comment]
(You need to run `./build indexes` to rebuild the search index, if you're wondering.)

**Comment by Zarel** [comment]
This is not what I mean by remove...

**Comment by Zarel** [comment]
https://github.com/Zarel/Pokemon-Showdown-Client/blob/9bf108dad87fbf77a26578bad3d94c2906a67cd6/js/search.js#L825-L848

**Comment by Zarel** [comment]
This is the slicing; you should slice at `slices["Restricted Legendary"]` I think

**Comment by HoeenCoder** [comment]
Ah that makes sense

**Comment by Zarel** [comment]
@HoeenCoder bump?

**Comment by HoeenCoder** [comment]
Ah shit I forgot about this, can do.

File: build-tools/build-indexes [hunk 1]
```diff
@@ -352,7 +352,7 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 
 		const tierOrder = (() => {
 			if (isVGC) {
-				return ["Mythical", "Restricted Legendary", "Regular", "NFE", "LC"];
+				return ["Restricted Legendary", "Regular", "NFE", "LC"];
```
Comment (Zarel) [inline build-tools/build-indexes]: This should be reverted:

```suggestion
				return ["Mythical", "Restricted Legendary", "Regular", "NFE", "LC"];
```

**Pull request merged**